docs/templates.rst

```diff
@@ -3,4 +3,90 @@
 Templates
 =========
 
-WRITE ME
+This document describes how to use templates with Django Bread.
+
+Template Context
+----------------
+
+In addition to what Django Vanilla Views provides, Django Bread provides the following to all of
```

Comment:
Let's add the word "context" somewhere in this sentence so it makes sense without reading the section title.